pkg/testcase/upgradenodereplacement.go

```diff
@@ -74,7 +78,8 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, version string) {
 
 func nodeReplaceAgents(
 	cluster *shared.Cluster,
-	version string,
+	version,
```

Comment:
have channel in same line?

Reply:
I think its better keep the vars in separate line